src/credential/sd_jwt_vc.rs

```diff
@@ -0,0 +1,169 @@
+use base64::prelude::*;
+use ssi::{
+    claims::{
+        jwt::{AnyClaims, JWTClaims},
+        sd_jwt::{RevealedSdJwt, SdJwtBuf},
+        vc::{v1::Credential as _, v2::Credential as _},
+        JwsString,
+    },
+    prelude::AnyJsonCredential,
+};
+use uuid::Uuid;
+
+use crate::{CredentialType, KeyAlias};
+
+use super::VcdmVersion;
+
+#[derive(uniffi::Object, Debug, Clone)]
+/// A verifiable credential secured as a JWT.
+pub struct SdJwtVc {
+    id: Uuid,
+    sd_jwt: JwsString,
+    credential: AnyJsonCredential,
+    credential_string: String,
+    header_json_string: String,
+    payload_json_string: String,
+    disclosures: Option<Vec<String>>,
```

Comment:
I think you are reimplementing `ssi::claims::sd_jwt::RevealedSdJwt`, which contains all this information, with stricter type info.

Reply:
Re-implemented type has been removed.